shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/uicases/offers/takeOffer/ReviewTradePresenter.kt

```diff
@@ -0,0 +1,36 @@
+package network.bisq.mobile.presentation.ui.uicases.offers.takeOffer
+
+
+import kotlinx.coroutines.flow.StateFlow
+import network.bisq.mobile.domain.data.model.OfferListItem
+import network.bisq.mobile.domain.service.offerbook.OfferbookServiceFacade
+import network.bisq.mobile.presentation.BasePresenter
+import network.bisq.mobile.presentation.MainPresenter
+import network.bisq.mobile.presentation.ui.navigation.Routes
+
+// TODO: Should do Interface for this?
```

Comment:
same as the other TODO. Please remove this TODOs after a decision has been made :)

Reply:
Done